substrate/frame/src/lib.rs

```diff
@@ -231,9 +236,19 @@ pub mod prelude {
 
 	/// Runtime traits
 	#[doc(no_inline)]
-	pub use sp_runtime::traits::{
-		BlockNumberProvider, Bounded, Convert, DispatchInfoOf, Dispatchable, ReduceBy,
-		ReplaceWithDefault, SaturatedConversion, Saturating, StaticLookup, TrailingZeroInput,
+	pub use sp_runtime::{
+		impl_tx_ext_default,
+		traits::{
+			AsSystemOriginSigner, AsTransactionAuthorizedOrigin, BlockNumberProvider, Bounded,
+			Convert, DispatchInfoOf, DispatchTransaction, Dispatchable, ReduceBy,
+			ReplaceWithDefault, SaturatedConversion, Saturating, StaticLookup, TrailingZeroInput,
+			TransactionExtension,
+		},
+		transaction_validity::{
+			InvalidTransaction, TransactionPriority, TransactionValidityError, UnknownTransaction,
+			ValidTransaction,
+		},
+		DispatchResult,
 	};
 	/// Other error/result types for runtime
 	#[doc(no_inline)]
```

Comment:
Can you please move the import from below to the other import above.

Reply:
It was the wrong import